src/graphinfo.jl

```diff
@@ -20,11 +20,11 @@ adjacency matrix and topologically ordered vertex list and stored.
 GraphInfo is instantiated using the `Model` constctor. 
 """
 
-struct GraphInfo{T} <: AbstractModelTrace
-    input::NamedTuple{T}
-    value::NamedTuple{T}
-    eval::NamedTuple{T}
-    kind::NamedTuple{T}
+struct GraphInfo{names_T, input_T, value_T, eval_T, kind_T} <: AbstractModelTrace
```

Comment:
Type names conventionally start with uppercase. `Tnames` etc. would be more idiomatic.